csrc/python_frontend/fusion_definition.cpp

```diff
@@ -151,11 +151,18 @@ void FusionDefinition::print(std::ostream& os) const {
 std::vector<at::Tensor> FusionDefinition::execute(
     const at::ArrayRef<c10::IValue>& inputs,
     bool override_user_schedule,
-    std::optional<int8_t> selected_device) const {
+    bool capture_debug_output,
+    std::optional<int8_t> selected_device) {
```

Comment:
nitpick, instead of dropping const here, can we make debug_output_ mutable? I think that's the only thing changed and it getting updated.

Reply:
Yeah `debug_output_` is the only part of `FusionDefinition` being updated. But how would I update it without dropping const?

Reply:
I like it. Will do. Thanks